templates/pages/setup/mailcollector/results_title.html.twig

```diff
@@ -0,0 +1,48 @@
+{#
+ # ---------------------------------------------------------------------
+ #
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ #
+ # http://glpi-project.org
+ #
+ # @copyright 2015-2023 Teclib' and contributors.
+ # @copyright 2003-2014 by the INDEPNET Development Team.
+ # @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # This program is free software: you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation, either version 3 of the License, or
+ # (at your option) any later version.
+ #
+ # This program is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+
+<div class="btn-group flex-wrap mb-3">
+   {% if servers|length > 0 %}
+      <span class="border-danger text-danger border-1 border p-0 ps-2 rounded-start">
+         <i class="ti ti-alert-triangle me-2"></i>
+         {% set server_links = servers|map((v) => '<a class="btn btn-ghost-danger" href="' ~ v['link'] ~ '">' ~ v['name'] ~ '</a>')|join(' ') %}
```

Comment:
Name should be escaped to prevent XSS.

```suggestion
         {% set server_links = servers|map((v) => '<a class="btn btn-ghost-danger" href="' ~ v['link'] ~ '">' ~ v['name']|e ~ '</a>')|join(' ') %}
```